### ISSUE TITLE
As a SO / SE I would like to select the article slider display style

### ISSUE THREAD

**Issue by ekellagher** [CuBoulder/express#2138]
The article slider display changed in 2.8.5 from title and summary displayed under the image thumbnail to title overlaid on top of the thumbnail.

This display does not work well when articles have long titles and the block is placed in the below content region and there is also content in the right sidebar
![industry site article slider](https://user-images.githubusercontent.com/16137919/38388030-34fee718-38d7-11e8-9da9-56f0255b159b.png)

It would be nice if the block had display choices such as the classic Title and Summary and the Title overlay to allow for more flexibility.

Related to Jira Issue:
https://cuboulder.atlassian.net/browse/EXP-9355
